churros/src/main/java/com/asapp/churros/IntExtensions.kt

```diff
@@ -0,0 +1,13 @@
+package com.asapp.churros
+
+fun Int.toOrdinalNumeral(): String {
```

Comment:
I'm worried about the localization here. I see we can use something like below, but only for API24+:
```
    import android.icu.text.MessageFormat

    private fun toOrdinal(number: Int): String {
        val formatter = MessageFormat("{0,ordinal}", Locale.getDefault())
        return formatter.format(arrayOf(number))
    }
```
How about we do the above, when possible?